extensions/functions/labelImage/labelImage.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//nolint:typecheck
```

Comment:
Why ignore these 3?

Reply:
the `github.com/mattn/go-tflite` pkg seems to be depended on CGO, which causes error like `xxx.h file not found`.

Reply:
reduced the number of nolint annotation to one.